packages/ketcher-core/src/domain/entities/text.ts

```diff
@@ -28,8 +28,16 @@ export enum TextCommand {
 export interface TextAttributes {
   // TODO: add Interface for content type
   content: string
-  position: Vec2
-  pos: Array<Vec2>
+  position: Point
+  pos: Array<Point>
+}
+
+function preparePositions(positions) {
```

Comment:
Probably, it would be good to add types:

function preparePositions(positions:Array<Point>) {

Reply:
Fix, thank you!